src/components/GamePage.vue

```diff
@@ -158,85 +206,21 @@ const KeyUp = () => {
   }
 };
 
-//中身をrefresh
-const refreshes = () => {
-  inputs.value = ""
-}
+//ページ全体を開いている時にどこを押してもkeyeventが起こる
+document.onkeydown = event => KeyDown();
+document.onkeyup = event => KeyUp();
```

Comment:
```suggestion
document.onkeydown = (event: KeyboardEvent) => KeyDown(event);
document.onkeyup = (event: KeyboardEvent) => KeyUp(event);
```
引数を渡さないと、KeyboardEvent ではなく Event になります